src/shilpa/Shop.java

```diff
@@ -0,0 +1,174 @@
+//Assignment  7  : 30th March 2021 
+
+//Create a system (Shop) which maintains each item's quantity.
+
+package shilpa;
+
+public class Shop {
+	int maggieQuantity, dosaQuantity, oilQuantity, panipuriQuantity, masalaQuantity;
+	
+	void Inventory(int maggie,int dosa, int oil,int panipuri,int masala ) {
+		maggieQuantity = maggie;
+		dosaQuantity = dosa;
+		oilQuantity = oil;
+		panipuriQuantity = panipuri;
+		masalaQuantity = masala;	
+	}
+	
+	void customer(int maggieOrder,int dosaOrder, int oilOrder,int panipuriOrder,int masalaOrder) {
+		if(maggieQuantity == 0) 
+		{
+			System.out.println("Maggie Packets running out of stock");
```

Comment:
Why we printing the OOS statements here, we have separate requirement for this FR2 and FR3